### ISSUE TITLE
Probleme with command lines

### ISSUE THREAD

**Issue by rpostolov** [symfony/symfony#15401]
Hello everyone !

Just for information...

I'm on symfony 2.7.2 and i use GeneratorBundle.

I have some issue qith this bundle when i run the commande line 

```php app/console admin:generate-admin```

Here is the what it return me : 

```Generator to use (doctrine, doctrine_odm, propel)
  [0] doctrine
  [1] doctrine_odm
  [2] propel```

But when i select 0 or doctrine i have again and again the same proposition.

With Symfony 2.7.1 everything is ok.

Thanks.

**Comment by WouterJ** [comment]
I think that this is more a bug in https://github.com/symfony2admingenerator/AdmingeneratorGeneratorBundle than in Symfony.

Btw, this bundle is based on some very old Symfony features (dialog helper is replaced in 2.5 by a question helper).

**Comment by mvhirsch** [comment]
@WouterJ I'm afraid to say: I can reproduce the exact same problem while using a Command written by myself. So it's a **bug**.

While turning on *error_reporting* and *display_errors*, you will get a *Notice: Undefined index: foo* on console output.
The problem occurs in the *QuestionHelper* of symfony and only happens at version 2.7.2.
While stepping back to version 2.7.1 I'm unable to reproduce the problem.

Steps to always reproduce:

```
$helper = $this->getHelper('question');
$question = new ChoiceQuestion('Name: ', array('foo', 'bar'), 0);

$name = $helper->ask($input, $output, $question);
```

I'll dig deeper, maybe I can spot the problem and provide a PR.

**Comment by mvhirsch** [comment]
Fix was already supplied with commit https://github.com/symfony/symfony/commit/3fcf61e664ccb12e9dd8ae98b33418cbfd095460

Will this fix be released in 2.7.3?

**Comment by WouterJ** [comment]
Yes, it will be included in 2.7.3. Didn't realize this was the same bug as #15343.

So this can be closed now

**Comment by mvhirsch** [comment]
Thank you very much!
